api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/color.md

```diff
@@ -8,4 +8,119 @@ default: undefined
 Specifies the color of the series in the hovered state.
```

Comment:
You don't want to use an article with words like "state", "mode", or "format". 

```suggestion
Specifies the color of the series in hovered state.
```

Reply:
Noted